extensions/exposition/components/identity.federation/manifest.toa.yaml

```diff
@@ -44,6 +44,10 @@ operations:
     output:
       identity:
         id: string
+        claim:
+          iss: string
+          aud: string
+          sub: string
```

Comment:
Don't we want to start using JSON schema for declarations?

Reply:
Yes, but next time.